packages/abbr/src/abbr.test.ts

```diff
@@ -1,7 +1,7 @@
 import { describe, test } from 'node:test'
 import assert from 'node:assert'
 
-import abbr from './abbr'
+const abbr = require('./abbr')
```

Comment:
```suggestion
import abbr = require('./abbr')
```
or maybe add a comment explaining why not use `import` here